precompile/contract/interfaces.go

```diff
@@ -33,7 +33,7 @@ type StateDB interface {
 	Exist(common.Address) bool
 
 	AddLog(addr common.Address, topics []common.Hash, data []byte, blockNumber uint64)
-	GetLogData() [][]byte
+	GetLogData() ([][]common.Hash, [][]byte)
```

Comment:
nit again consider naming parameters (topics and logData maybe)

Reply:
added